additional/01-howtos/use-menditect-extension.md

```diff
@@ -0,0 +1,85 @@
+# Use Menditect Extension in Studio Pro
+
+:::caution
+The Menditect Call Hierarchy extension is currently in BETA.
+:::
+
+## Howto
+
+This document describes how to import and configure the [Menditect Call Hierarchy extension <i class="fas fa-external-link"></i>](https://marketplace.mendix.com/link/component/xxx), so you can use it in Mendix Studio Pro.
+
+The Menditect Call Hierarchy extension can be used on a microflow in Mendix Studio Pro, to recusively view all submicroflows that are called from that microflow in a tree structure.
+
+The Menditect Call Hierarchy extension can also be used to [generate a microflow test](generate-test#test-a-microflow), and view [test cases](../../test-case) where that microflow is executed.
+
+
+## Installation
+
+To install the Menditect Call Hierarchy extension:
+- Download [Mendix Studio Pro 10.6.3 <i class="fas fa-external-link"></i>](https://marketplace.mendix.com/link/studiopro/10.6.3) or higher;
+- Create or open a Mendix App in this version of Studio Pro;
+- Download the [Menditect Call Hierarchy extension <i class="fas fa-external-link"></i>](https://marketplace.mendix.com/link/component/xxx) from the Mendix marketplace;
+- Import it in the Mendix App.
+
+## Microflow Call Hierarchy
+
+To view the Microflow Call Hierarchy:
+- Make sure the App Explorer tab is visible in Studio Pro.
+- Right-click on any Microflow in the App Explorer.
+- Select the "Menditect" sub menu.
+- Click on "Microflow Call Hierarchy".
+
+Mendix will open the Microflow Call Hierarchy tab and show all submicroflows called from the selected microflow, recursively.
+
+Double-click on any of the microflows in this tab, to open it in Studio Pro.
+
+
+## MTA integration
+
+### Prerequisites
+
+In order to use the MTA integration features of the Extension, some information needs to be entered in Studio Pro.
+
```

Comment:
Add the requirement that there is a testconfiguration in MTA which contains the current Mendix project